File: app/src/main/java/com/bopr/android/smailer/ui/TextWhitelistActivity.java
package com.bopr.android.smailer.ui;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

/**
 * Text whitelist activity.
 *
 * @author Boris Pronin (<a href="mailto:boprsoft.dev@gmail.com">boprsoft.dev@gmail.com</a>)
 */
public class TextWhitelistActivity extends AppActivity {

    @NonNull
    @Override
    protected Fragment createFragment() {
        return new TextWhitelistFragment();
    }

}

File: app/src/main/kotlin/com/bopr/android/smailer/ui/TextWhitelistActivity.kt
package com.bopr.android.smailer.ui

import androidx.fragment.app.Fragment

/**
 * Text whitelist activity.
 *
 * @author Boris Pronin ([boprsoft.dev@gmail.com](mailto:boprsoft.dev@gmail.com))
 */
class TextWhitelistActivity : AppActivity() {

    override fun createFragment(): Fragment {
        return TextWhitelistFragment()
    }
}